core/src/restart_overlay.cpp

```diff
@@ -0,0 +1,46 @@
+#include "qabstractbutton.h"
+
+#include <QCoreApplication>
+#include <QDesktopWidget>
+#include <QFile>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QStyle>
+#include <QTextBrowser>
+
+#include <restart_overlay.h>
+#include <pluginbase/preferences.h>
+#include <stylehelper.h>
+#include <ui_licensedialogoverlay.h>
+#include <application_restarter.h>
+using namespace scopy;
+
+RestartOverlay::RestartOverlay(QWidget *parent)
+	: QDialog(parent)
+	, parent(parent)
+{
+	ui = new Ui::LicenseDialogOverlay();
```

Comment:
This file should be renamed in another PR. I think this could also be refactored into an class without an .ui file.